docker/main/ngen/Dockerfile

```diff
@@ -666,8 +670,9 @@ RUN if [ "${NGEN_WITH_PYTHON}" == "ON" ]; then \
 USER ${USER}
 
 ENV VIRTUAL_ENV=/dmod/venv
-RUN python3 -m venv $VIRTUAL_ENV && pip3 install numpy
+RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN if [ "${NGEN_WITH_PYTHON}" == "ON" ]; then pip install numpy${NUMPY_CONSTRAINT} ; fi
```

Comment:
This is an abuse of the args parameter. Do not use this to inject python versions. Stick to setup.py and requirements.txt.